sklearn/tests/test_common.py

```diff
@@ -68,6 +72,21 @@ def _tested_estimators():
                     estimator = Estimator(Ridge())
                 else:
                     estimator = Estimator(LinearDiscriminantAnalysis())
+            elif "estimators" in required_parameters:
+                if issubclass(Estimator, RegressorMixin):
```

Comment:
Nit: I know it's from the original code above, but we could use `is_classifier` and `is_regressor`?

Reply:
`is_regressor` and `is_classifier` will expect an instance while `Estimator` is the class.
I think that this is the reason.